composer.json

```diff
@@ -15,19 +15,19 @@
     "require": {
         "php": "^5.3.3 || ^7.0",
         "twig/twig": "^1.27 || ^2.0",
-        "symfony/framework-bundle": "^2.3 || ^3.0",
+        "symfony/framework-bundle": "^2.3 || ^3.0 || ^4.0",
         "nikic/php-parser": "^1.4 || ^2.0 || ^3.0",
-        "symfony/console": "^2.3 || ^3.0"
+        "symfony/console": "^2.3 || ^3.0 || ^4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.8.27",
+        "phpunit/phpunit": "^5.4",
```

Comment:
How about using `symfony/phpunit-bridge` instead? This will avoid yaml-related issues. I tried it this morning, and it worked, but I didn't have the time to make a PR. Will make one on your fork unless you beat me to it.

Reply:
Sorry no idea on how to properly configure this. Most of the tests now fail where previously only a couple of them did.

Reply:
I did it in my PR, give it a look: https://github.com/mvrhov/JMSTranslationBundle/pull/1

Reply:
thanks